Anilist4Net.Test/MediaTests.cs

```diff
@@ -95,5 +95,15 @@ public async Task HigurashiTest()
 			IsNotNull(media.StartDate.ToDate());
 			AreEqual(new DateTime(2009, 1, 1, 0, 0, 0, DateTimeKind.Unspecified), media.StartDate.ToDate());
 		}
+
+		// Caters for cases where there are more than one Mal Id that is the same (for example an anime and a manga) by allowing the user to specify the media type they expect
+        [TestCase(MediaTypes.ANIME, "Yakusoku no Neverland 2")]
+        [TestCase(MediaTypes.MANGA, "Vermillion")]
+		public async Task SettingExpectedMediaTypeCorrectlyDifferentiateMatchingMalIds(MediaTypes typeToRetrieve, string expectedTitle)
```

Comment:
```suggestion
		public async Task MalIdByTypeTest(MediaTypes typeToRetrieve, string expectedTitle)
```
The test name here is a little long winded, but apart from that this PR looks good :)